modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -567,14 +567,20 @@
 		</annotation>
 		<annotation name="@rpc" qualifiers="vararg">
 			<return type="void" />
-			<param index="0" name="mode" type="String" default="&quot;&quot;" />
-			<param index="1" name="sync" type="String" default="&quot;&quot;" />
-			<param index="2" name="transfer_mode" type="String" default="&quot;&quot;" />
+			<param index="0" name="mode" type="String" default="&quot;authority&quot;" />
+			<param index="1" name="sync" type="String" default="&quot;call_remote&quot;" />
+			<param index="2" name="transfer_mode" type="String" default="&quot;unreliable&quot;" />
```

Comment:
Defaults are generated from the code, you can't edit them by hand like that. This needs to be reverted.